Contrib/MLpackages/GlobalAncestry.py

```diff
@@ -0,0 +1,147 @@
+#Global Ancestry with 1k genomes
+
+import pandas as pd
+import numpy as np
+import torch
+import sys
+
+from sklearn import datasets, linear_model
+from sklearn.metrics import mean_squared_error, r2_score
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.linear_model import LinearRegression
+
+#Load the vcf file and ancestry information file
+#genofile = '1kGPfiltered/pruned500k1kGP.vcf'
+#AncstFile = 'igsr_IDwAncst_subset-Copy1.txt'
+#AncstFile = 'igsr_IDwAncst.txt'
+genofile = open(sys.argv[1],'r')
+AncstFile = open(sys.argv[2],'r')
+log = open(sys.argv[3],'w')
+filename = sys.argv[4]
+
+
+
+#Read in both files
+headervcf=pd.read_csv(genofile,delimiter='\t',skiprows=25)
+IDswAncst = pd.read_csv(AncstFile,delimiter = ',')
+
+
+
+#Turn genotypes into dosage format for PCA
+dosagevcf = headervcf.replace('0/0', 0)
+dosagevcf = dosagevcf.replace('0/1', 1)
+dosagevcf = dosagevcf.replace('1/0', 1)
+dosagevcf = dosagevcf.replace('1/1', 2)
```

Comment:
Also, we're applying these `replace` transformations over the whole file, but we only intend for them to have effect in certain columns.  It might be clearer in that case to specify the cols where we expect this replace to apply, which would help with legibility and reduce the risk of a solution that works by accident.  Off the top of my head I couldn't be sure that no other string in the df contains a substring of the form '{0,1}/{0,1}'.